### PR TITLE
update theme Inside screenshot and meta

### PR DESCRIPTION
**Pull request by elmorec**
- [x] Read the [theme publishing doc](https://hexo.io/docs/themes#Publishing) or [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).

### CODE REVIEW

File: source/_data/themes.yml [hunk 1]
```diff
@@ -437,14 +437,16 @@
     - light
     - colorful
 - name: Inside
-  description: Flat and clean theme ❤️
+  description: ❤️ SPA, flat and clean theme for Hexo.
```
Comment (YoshinoriN) [inline source/_data/themes.yml]: We homogenized description in #899 
Could you please delete `for Hexo`?

File: source/_data/themes.yml [hunk 2]
```diff
@@ -437,14 +437,16 @@
...
   link: https://github.com/elmorec/hexo-theme-inside
   preview: https://blog.oniuo.com/
   tags:
+    - responsive
     - flat
     - clean
-    - SPA
-    - Angular
+    - spa
+    - pwa
```
Comment (YoshinoriN) [inline source/_data/themes.yml]: Please use uppercase `spa` & `pwa`.
Like `SPA`, `PWA`

File: source/_data/themes.yml (hunk 1 above)
Comment (elmorec) [reply source/_data/themes.yml]:
```suggestion
  description: ❤️ SPA, flat and clean theme.
```

File: source/_data/themes.yml (hunk 2 above)
Comment (elmorec) [reply source/_data/themes.yml]:
```suggestion
    - PWA
```

File: source/_data/themes.yml [hunk 3]
```diff
@@ -437,14 +437,16 @@
...
+  description: ❤️ SPA, flat and clean theme for Hexo.
   link: https://github.com/elmorec/hexo-theme-inside
   preview: https://blog.oniuo.com/
   tags:
+    - responsive
     - flat
     - clean
-    - SPA
-    - Angular
+    - spa
```
Comment (elmorec) [inline source/_data/themes.yml]:
```suggestion
    - SPA
```

**Comment by YoshinoriN** [comment]
Thanks!
Merged :)

**Pull request merged**